### DISCUSSION TITLE
Erreur de typage pour text[]

### DISCUSSION THREAD

**Discussion by doriangi** [amt-classroom/amt-classroom.github.io#3] [Q&A]
Erreur : 'Basic' attribute type should not be 'String[]' pour le champ special_feature de la table film

**Comment by Perdjesk** [comment]
Voici deux références qui mettent sur la voie de en tout cas deux approches pour résoudre ce mapping.

https://www.baeldung.com/java-hibernate-map-postgresql-array
https://docs.jboss.org/hibernate/orm/6.2/userguide/html_single/Hibernate_User_Guide.html#basic-custom-type
